docs/authentication-authorization/authentication/google-account.md

```diff
@@ -38,15 +38,20 @@ in the Klaw Core module to enable the use of Google account login.
 
     `klaw.superadmin.default.username=superadmin@company.com`
 
-6.  If you have already signed up with Google, you will be redirected to the Klaw home page.
+6.  Each SSO provider can have a different "preferred username" attribute which identifies your user correctly.
+    For Google update this attribute in the application.properties as below.
+
+    `klaw.ad.username.attribute=email`
```

Comment:
We should not force the organization to update this claim, as preferred_username can also be present in google SSO claims.